yarn.lock

```diff
@@ -331,7 +331,7 @@
     "@babel/helper-plugin-utils" "^7.13.0"
     "@babel/plugin-syntax-logical-assignment-operators" "^7.10.4"
 
-"@babel/plugin-proposal-nullish-coalescing-operator@^7.13.0", "@babel/plugin-proposal-nullish-coalescing-operator@^7.13.8":
+"@babel/plugin-proposal-nullish-coalescing-operator@^7.13.8":
```

Comment:
FYI: This was the result after I removed the `codecov` package with `yarn remove codecov`. I checked for what packages are requiring `@babel/plugin-proposal-nullish-coalescing-operator`, and it's only `@babel/preset-env` which does in fact request `@babel/plugin-proposal-nullish-coalescing-operator@^7.13.8`.

So I believe what we see here is correct, and the fact that there were two versions of this package previously installed was unnecessary and that something had gotten out of sync with the lock file.